ext/soap/tests/classmap006.phpt

```diff
@@ -0,0 +1,59 @@
+--TEST--
+SOAP Classmap 6: encoding of inherited objects with namespace
+--EXTENSIONS--
+soap
+--FILE--
+<?php
+ini_set("soap.wsdl_cache_enabled",0);
+
+class A {
+  public $x;
+  function __construct($a){
+    $this->x = $a;
+  }
+}
+class Attest {
+  public $x;
+  function __construct($a){
+    $this->x = $a;
+  }
+}
+class B extends A {
+  public $y;
+  function __construct($a){
+    parent::__construct($a);
+    $this->y = $a + 1;
+  }
+}
+
+function f($input){
+  return new B(5);
+}
+
+class LocalSoapClient extends SoapClient {
+
```

Comment:
Add a field: `private $server;` to make the test work on 8.2 and above, it avoids the deprecation warning about dynamic properties.